### PR TITLE
Feature/861 batch actions modal

### PR DESCRIPTION
**Pull request by chris-lawton**
branched from `feature/860-batch-review`

- adds updated modal styles

### CODE REVIEW

File: opentech/static_src/src/javascript/apply/batch-actions.js [hunk 1]
```diff
@@ -47,18 +50,46 @@
...
         let allids = [];  // create an array of submission id's from the selected checkboxes
 
         $checkbox.each(function () {
             if ($(this).is(':checked')) {
-                $batchTitles.append(`<p>${$(this).parents('tr').find('.js-title').data('tooltip')}</p>`);
+                const href = $(this).parents('tr').find('.js-title').find('a').attr('href');
+                const title = $(this).parents('tr').find('.js-title').data('tooltip');
+
+                $batchTitlesList.append(`
+                    <a href="${href}" class="modal__list-item" target="_blank" rel="noopener noreferrer">
```
Comment (helenb) [inline opentech/static_src/src/javascript/apply/batch-actions.js]: add title to a tag

File: opentech/static_src/src/sass/apply/fancybox.scss [hunk 2]
```diff
@@ -424,6 +428,13 @@ body.fancybox-iosfix {
 	transition: background-color .25s;
 	box-sizing: border-box;
 	z-index: 2;
+
+	.modal--secondary & {
+		color: white;
+		font-size: 40px;
+		margin-top: 12px;
+    	margin-right: 8px;
```
Comment (helenb) [inline opentech/static_src/src/sass/apply/fancybox.scss]: indentation

File: opentech/static_src/src/javascript/apply/batch-actions.js (hunk 1 above)
Comment (chris-lawton) [reply opentech/static_src/src/javascript/apply/batch-actions.js]: 👍

File: opentech/static_src/src/sass/apply/fancybox.scss (hunk 2 above)
Comment (chris-lawton) [reply opentech/static_src/src/sass/apply/fancybox.scss]: weird, that's not showing up in my editor.

**Pull request merged**